Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```
@@ -498,8 +498,8 @@ div.bocList div.bocListTableBlock div.bocListTableContainer div.bocListFakeTable
 div.bocList div.bocListTableBlock div.bocListTableContainer div.bocListFakeTableHead > div
 {
   /* Reserve space in case there is a scrollbar for the data-part of the table.
-     Note that the actual max-width is not important so long as the scrollbar's max-width is covered. '*/
-  border-right: var(--remotion-themed-scrollbar-width) solid var(--boclist-tablehead-background-color);
+    Note that the actual max-width is not important so long as the scrollbar's max-width is covered. '*/
```

```suggestion
     Note that the actual max-width is not important so long as the scrollbar's max-width is covered. '*/
```
one more whitespace to reset the changes of this line in the PR :)